data/json/itemgroups/Clothing_Gear/clothing.json

```diff
@@ -4517,10 +4517,14 @@
   {
     "type": "item_group",
     "id": "aspirin_bottle_plastic_pill_painkiller_20",
```

Comment:
I wonder if it's worth doing a find-and-replace for this and renaming it "otc_painkiller" or similar instead.

Reply:
I have this done, but I won't push the change because it affects 20+ files and would screw up your review progress. Once you're done I'll push it.

Reply:
Pushed now.